libs/api/domains/license-service/src/lib/licenceService.type.ts

```diff
@@ -0,0 +1,101 @@
+// export type LICENSE_TYPE = 'DRIVERS_LICENSE' // | 'HUNTING_LICENSE'
+
+export enum GenericLicenseType {
+  DriversLicense = 'DriversLicense',
+  HuntingLicense = 'HuntingLicense',
+}
+
+export enum GenericLicenseProviderId {
+  NationalPoliceCommissioner = 'NationalPoliceCommissioner',
+}
+
+export enum GenericUserLicenseStatus {
+  Unknown = 'Unknown',
+  HasLicense = 'HasLicense',
+  NotAvailable = 'NotAvailable',
+}
+
+export enum GenericUserLicenseFetchStatus {
+  Fetched = 'Fetched',
+  NotFetched = 'NotFetched',
+  Fetching = 'Fetching',
+  Error = 'Error',
+  Stale = 'Stale',
+}
+
+export enum GenericLicenseDataFieldType {
+  Group = 'Group',
+  Category = 'Category',
+  Value = 'Value',
+}
+
+/*
+export const LICENSE_TITLES: Record<LICENSE_TYPE, string> = {
+  DRIVERS_LICENSE: 'Ökuskírteini',
+  // HUNTING_LICENSE: 'Veiðikort',
+}
+
+export const LICENSE_PROVIDERS: Record<LICENSE_TYPE, keyof typeof PROVIDERS> = {
+  DRIVERS_LICENSE: 'national-police-commissioner',
+  // HUNTING_LICENSE: 'environment-agency',
+}
+*/
+
+export type GenericLicenseProvider = {
+  id: GenericLicenseProviderId
+
+  // TODO(osk) should these be here? or be resolved by client via contentful?
+  // Commented out until talked about, to limit scope of v1
+  /*
+  name: string
+  logo?: string
+  */
+}
+
+export type GenericLicenseMetadata = {
+  type: GenericLicenseType
+  provider: GenericLicenseProvider
+  pkpass: boolean
+  timeout: number
+
+  // TODO(osk) should these be here? or be resolved by client via contentful?
+  // Commented out until talked about, to limit scope of v1
+  /*
+  title: string
+  ordering: number
+  backgroundImage?: string
+  applicationUrl?: string
+  detailUrl?: string
+  */
```

Comment:
Let's make sure we get rid of all commented code asap.

Reply:
yes! starting a new branch right away